decidim-bulletin_board-js/src/trustee/__mocks__/trustee.js

```diff
@@ -1,6 +1,13 @@
+import { buildMessageId } from "../../test-utils";
+
 export class Trustee {
   processLogEntry({ messageId, signedData }) {
-    switch (messageId) {
+    const typeSubtype = messageId
+      .split("+")[0]
+      .split(".", 4)
+      .slice(2, 4)
+      .join(".");
```

Comment:
I am a bit worried about having these implementation details here. Maybe we can create a function in the `test-utils` file?